src/Magento/FunctionalTestingFramework/Util/TestGenerator.php

```diff
@@ -1642,6 +1626,69 @@ private function addUniquenessToParamArray($input)
         return implode(", ", $result);
     }
 
+    /**
+     * Process pressKey parameterArray attribute for uniqueness function call and necessary data resolutions
+     *
+     * @param string $input
+     * @return string
+     */
+    private function processPressKey($input)
+    {
+        // validate the param array is in the correct format
+        $input = trim($input);
+        $this->validateParameterArray($input);
+        // trim off the outer braces
+        $input = substr($input, 1, strlen($input) - 2);
+
+        $result = [];
+        $arrayResult = [];
+        $count = 0;
+
+        // matches arrays
```

Comment:
I think we need a little more commenting on here, what's actually happening is more like
`find arrays, replace them with placeholder to prevent manipulation in foreach below`